pkg/util/kube/utils.go

```diff
@@ -187,6 +196,23 @@ func isProvisionedByCSI(log logrus.FieldLogger, pv *corev1api.PersistentVolume,
 	return false, nil
 }
 
+// SinglePathMatch function will be called by PVB and PVB controller to check whether pass-in volume path is valid.
+// Check whether there is only one match by the path's pattern (/host_pods/%s/volumes/*/volume_name/[mount|]).
+func SinglePathMatch(path string, fs filesystem.Interface, log logrus.FieldLogger) (string, error) {
+	matches, err := fs.Glob(path)
+	if err != nil {
+		log.Errorf("fail to search path by pattern %s: %s", path, err.Error())
```

Comment:
Don't log the error, just return it. Or the log will be duplicated

Reply:
Removed.